todo/src/components/Todo.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { connect } from "react-redux";
+
+import { addTask, toggleDone } from "../actions";
+
+class ToDoList extends React.Component {
+    state = {
+        newTodo: ""
+    };
+    handleChanges = e => {
+        this.setState({ newTodo: e.target.value});
+    };
+    addTask = e => {
+        e.preventDefault();
+        this.props.addTask(this.state.addTask);
+    };
+    toggleDone = (e, index) => {
+        this.props.toggleDone(index);
+    };
+    render(){
+        return (
+            <React.Fragment>
+            <div className = "todo-list">
+                {this.props.tasks.map((task, index)=> (
+                    <h4 onClick={e => this.toggleDone(e, index)} key={index}>
+                        {task.name}
+                        {task.done && <i className="fas fa-dragon" />}
```

Comment:
you'll want to toggle a strikethrough on your todo - just like you did for your react project. The dragon icon is from the guided project.